installing/installation-guide.md

```diff
@@ -28,37 +28,11 @@ Therefore, we *strongly* recommended that, prior to starting the Qubes installer
 Furthermore, if you are installing Qubes on a potentially compromised system, we *strongly* recommended that you wipe your target installation disk before starting the installer.
 
 
-Qubes 4.0 Warning
------------------
-
-In new installations of Qubes 4.0, the following steps may need to be applied in dom0 and Fedora 26 TemplateVMs in order to receive updates (see [#3737]).
-
-Steps for dom0 updates:
-
-1. Open the Qubes Menu by clicking on the "Q" icon in the top-left corner of the screen.
-2. Select `Terminal Emulator`.
-3. In the window that opens, enter this command:
-
-       sudo nano /etc/yum.repos.d/qubes-dom0.repo
-
-4. This opens the nano text editor. Change all four instances of `http` to `https`.
-5. Press `CTRL+X`, then `Y`, then `ENTER` to save changes and exit.
-6. Check for updates normally.
-
-Steps for Fedora 26 TemplateVM updates:
-
-1. Open the Qubes Menu by clicking on the "Q" icon in the top-left corner of the screen.
-2. Select `Template: fedora-26`, then `fedora-26: Terminal`.
-3. In the window that opens, enter the command for your version:
-
-       [Qubes 3.2] sudo gedit /etc/yum.repos.d/qubes-r3.repo
-       [Qubes 4.0] sudo gedit /etc/yum.repos.d/qubes-r4.repo
+Qubes 4.0.1 Warning
+-------------------
 
-4. This opens the gedit text editor in a window. Change all four instances of `http` to `https`.
-5. Click the "Save" button in the top-right corner of the window.
-6. Close the window.
-7. Check for updates normally.
-8. Shut down the TemplateVM.
+After installing Qubes 4.0.1, please install fresh APT-based (e.g., Debian and Whonix) TemplateVMs in order to be protected from the APT update mechanism vulnerability that was patched after the release of 4.0.1.
+For instructions and further details, please see [QSB #46].
```

Comment:
Since just after installation templates are definitely installed using rpm (not for example upgraded from previous template version or cloned), there is an easier option to reinstall them:
```
sudo qubes-dom0-update --action=upgrade qubes-template-debian-9
```
(and similar for other templates)

Do you think it worth noting it here? The full procedure described in the QSB of course works too, but is more complex (to handle the cases where template wasn't installed directly with rpm).

Reply:
Sure, let's make it easier on users.

Reply:
Let me know what you think of the updated recommendation, @marmarek.